packages/helm-charts/blockscout/templates/blockscout-web.deployment.yaml

```diff
@@ -113,23 +116,19 @@ spec:
         - name: LEARNING_MENU_LIST
           value: {{ .Values.blockscout.web.appsMenu.learningList | quote }}
         - name: RE_CAPTCHA_PROJECT_ID
-          valueFrom:
-            secretKeyRef:
-              name:  {{ .Values.blockscout.web.recaptchaSecretName | quote }}
-              key: project_id
+          value: {{ .Values.blockscout.secrets.recaptcha.projectId }}
```

Comment:
Q for my understanding: this was previously a single k8s secret which was essentially a map of 3 values indexed by the `key` to retrieve the necessary credential. Are these now individual secrets within gcp's credential management system?

Reply:
Yes, that's right, there are three different secrets in GCP Secret Manger. I thought we could ditch `projectId` and `siteKey` as they are rendered in open text anyway but left for completeness.